src/generator/generator.controller.js

```diff
@@ -188,6 +195,11 @@ function addPostRunCheckIn (plays) {
     return [...plays, new SpecialPlay('special:post-run-check-in', hosts, templates.special.postRunCheckIn)];
 }
 
+function addComplianceReportPlay (plays) {
+    const hosts = _(plays).flatMap('hosts').uniq().sort().value();
+    return [...plays, new SpecialPlay('special:generate-report', hosts, templates.compliance.generateReport)];
```

Comment:
Name of the special play might be `compliance:generate-report`.

Reply:
Ah yes, you're right. Thanks!